scripts/cli/tests/0_create_identities.js

```diff
@@ -49,18 +52,23 @@ async function createIdentities(api, accounts, alice) {
           .registerDid([])
           .signAndSend(
             accounts[i],
-            { nonce: reqImports["nonces"].get(accounts[i].address) },
+            { nonce: reqImports.nonces.get(accounts[i].address) },
             ({ events = [], status }) => {
+
               if (status.isFinalized) {
-                reqImports["fail_count"] = reqImports["callback"](status, events, "identity", "NewDid", reqImports["fail_count"]);
+                // Loop through Vec<EventRecord> to display all events
+                events.forEach(({ phase, event: { data, method, section } }) => {
+                  if ( section === "system" && method === "ExtrinsicSuccess" )  reqImports.fail_count--;
```

Comment:
Could we log `system.ExtrinsicFailed`?

Reply:
you mean `console.log` or store in a log file ?